include/revng/Pipes/Ranks.h

```diff
@@ -31,13 +31,14 @@ inline auto Function = defineRank<"function", model::Function::Key>(Binary);
 inline auto BasicBlock = defineRank<"basic-block", BasicBlockID>(Function);
 inline auto Instruction = defineRank<"instruction", MetaAddress>(BasicBlock);
 
-inline auto Type = defineRank<"type", model::Type::Key>(Binary);
+inline auto Type = defineRank<"type", model::TypeDefinition::Key>(Binary);
```

Comment:
Shall we rename the rank?
Maybe you're doing it in a later commit.

Reply:
Ah, right. Thanks for noticing.

Added the commit for now, let's see how much stuff it breaks.